Gaijinizer patch/Map041.txt

```diff
@@ -1299,92 +1381,97 @@ How much...I am Lucy ...#MTLed
 ちゃんと加減はしていたのですが、
 あなたが脱水症状になるまで、止められませんでした…
 > CONTEXT: Map041/events/5/pages/1/642/Dialogue/yuuma
-Is it about one hour? I was modest, but
-I could not stop until you got dehydrated ...#MTLed
+It was about an hour. I tried to stay moderate but
+I couldn't stop myself until you got dehydrated...#Ron
```

Comment:
Think we can rephrase the second sentence here a bit:
>_**It was about an hour. I tried to moderate myself,
but I couldn't stop until you got dehydrated...**_

(The first sentence is a bit stiff and overly formal too, but it seems like you're going with that for her style of tone.)

Reply:
Yeah, her Japanese is pretty stiff and overly polite which I'm translating to be "lady-like", something akin to a high-class Victorian lady.